test/test_utils/__init__.py

```diff
@@ -79,6 +80,18 @@ def run_subprocess_cmd(cmd, failure="Command failed"):
     return command
 
 
+def login_to_ecr_registry(context, account_id, region):
```

Comment:
Does it make sense to create a fixture for this, to be reused?

Reply:
The reason I made it a utility function is so that it could be used outside of pytest (i.e. for canary utilities, or other utilities that might involve logging into the ecr registry). For that reason, I think it makes more sense to keep it as a utility function

Reply:
Got it, can you replace following line be replaced to use the new login command? - https://github.com/aws/deep-learning-containers/blob/master/buildspec.yml#L12